unit-testing/presentation.html

```diff
@@ -1,27 +1,398 @@
+            maintain.</li>
+        </ul>
+      </section>
+      <!-- 8 slide -->
+      <section class="table-slide">
+        <h3 class="slide-name">TDD VS BDD</h3>
+        <table class="fragment" data-fragment-index="2">
+          <table data-fragment-index="2">
+            <thead>
+              <tr>
+                <th>TDD</th>
+                <th>BDD</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <p>TDD refers to the practice of writing a piece of code only if the automated test has failed</p>
+                </td>
+                <td>
+                  <p>BDD is a process that promotes collaboration between developers, testers and customer service
+                    personnel during product development</p>
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <p>Developers are the key participants in TDD</p>
+                </td>
+                <td>
+                  <p>Developers, customers and QAs are the key participants</p>
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <p>Mainly concentrates on unit tests</p>
+                </td>
+                <td>
+                  <p>Mainly concentrates on system requirements</p>
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <p>Point of inception is a test case</p>
+                </td>
+                <td>
+                  <p>Point of inception is a test scenario</p>
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <p>TDD is a development practice</p>
+                </td>
+                <td>
+                  <p>BDD is a team methodology</p>
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <p>Collaboration is limited to developers in TDD</p>
+                </td>
+                <td>
+                  <p>Collaboration is required by all stakeholders in BDD</p>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+      </section>
+      <!-- 9 slide -->
+      <section>
+        <h3 class="slide-name">TDD is RED-GREEN-BLUE development​</h3>
+        <img src='assets/images/tdd.png' height="500" />
+      </section>
+      <!-- 10 slide -->
+      <section>
+        <!-- 10/0 slide -->
+        <section>
+          <h3 class="slide-name">Testing statistic​</h3>
+          <img src='assets/images/statistic_1.png' />
+        </section>
+        <!-- 10/1 slide -->
+        <section>
+          <h3 class="slide-name">Testing statistic​</h3>
+          <img src='assets/images/statistic_2.png' />
+        </section>
+      </section>
+      <!-- 11 slide -->
+      <section>
```

Comment:
class="picture-with-description-slide" and please change order of text and image